Tests_v3/StreamChatClientTestPlan.xctestplan

```diff
@@ -1,32 +1,37 @@
 {
-  "configurations" : [
+  "configurations": [
     {
-      "id" : "5358EB81-7D4D-42F7-B281-016C02706988",
-      "name" : "Configuration 1",
-      "options" : {
-
+      "id": "5358EB81-7D4D-42F7-B281-016C02706988",
+      "name": "Configuration 1",
+      "options": {
       }
     }
   ],
-  "defaultOptions" : {
-    "codeCoverage" : {
-      "targets" : [
+  "defaultOptions": {
+    "codeCoverage": {
+      "targets": [
         {
-          "containerPath" : "container:StreamChat_v3.xcodeproj",
-          "identifier" : "799C941A247D2F80001F1104",
-          "name" : "StreamChatClient"
+          "containerPath": "container:StreamChat_v3.xcodeproj",
+          "identifier": "799C941A247D2F80001F1104",
+          "name": "StreamChatClient"
         }
       ]
-    }
+    },
+    "environmentVariableEntries": [
+      {
+        "key": "CI",
+        "value": "TRUE"
+      }
+    ]
   },
-  "testTargets" : [
+  "testTargets": [
     {
-      "target" : {
-        "containerPath" : "container:StreamChat_v3.xcodeproj",
-        "identifier" : "799C9450247D59B1001F1104",
-        "name" : "StreamChatClientTests"
+      "target": {
+        "containerPath": "container:StreamChat_v3.xcodeproj",
+        "identifier": "799C9450247D59B1001F1104",
+        "name": "StreamChatClientTests"
       }
     }
   ],
-  "version" : 1
-}
+  "version": 1
+}
```

Comment:
This shouldn't be here, it's just noise from running CI script locally.

Reply:
Yeah exactly I wrote in the description :D IMHO if this noise will be generated everytime test_v3 lane is run, why don't we commit it? Running a lane locally should not generate changes in repo

Reply:
Two things:
- when you edit the testplan in Xcode, it will revert all whitespace changes back
- you can see, it adds the `CI` flag there. You don't want to run it locally with the CI flag

Reply:
I think it's on our list in the Fastfile improvements JIRA ticket :)